.github/workflows/ios_ci.yaml

```diff
@@ -45,14 +45,21 @@ jobs:
       - uses: futureware-tech/simulator-action@v3
         id: simulator-action
         with:
-          model: 'iPhone 15'
+          model: "iPhone 15"
           shutdown_after_job: false
 
-  build-macos:
+  build-iOS:
     if: github.event.pull_request.head.repo.full_name != github.repository
     runs-on: macos-latest
 
     steps:
+      - name: Maximize build space
```

Comment:
It appears that this cleanup step doesn't work most of the time.

Normally, I execute this command: `gh cache delete --all --repo AppFlowy-IO/AppFlowy`. It has a better effect than this step when there's a 'no space left' error.